pkg/plugin/healthz.go

```diff
@@ -14,20 +14,24 @@ import (
 	"os"
 	"time"
 
+	"github.com/Azure/kubernetes-kms/pkg/utils"
 	"github.com/Azure/kubernetes-kms/pkg/version"
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 	pb "k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1"
```

Comment:
Also, change the import path to `k8s.io/kms/apis/v1beta1` in all the places (xref: https://github.com/kubernetes/kms/tree/master/apis/v1beta1)